recipes/eventio/conda_build_config.yaml

```diff
@@ -0,0 +1,7 @@
+python:
+  - 3.7
+  - 3.8
+
+numpy:
+  - 1.18
+  - 1.19
```

Comment:
You don't need this file. In conda-forge we will build for Pythons 3.6, 3.7, 3.8 and 3.9. For numpy we use the oldest one possible in the build, to ensure forward compatibility, and let it "run" with the newest available.